gensrc/script/doris_builtins_functions.py

```diff
@@ -2180,8 +2180,14 @@
     [['split_part'], 'VARCHAR', ['VARCHAR', 'VARCHAR', 'INT'],
         '_ZN5doris15StringFunctions10split_partEPN9doris_udf15FunctionContextERKNS1_9StringValES6_RKNS1_6IntValE',
         '', '', 'vec', 'ALWAYS_NULLABLE'],
+
      [['extract_url_parameter'], 'VARCHAR', ['VARCHAR', 'VARCHAR'],'','', '', 'vec', ''],
 
+    # runningdifference
+    [['running_difference'], 'INT', ['INT'],
+        '',
+        '', '', 'vec', 'ALWAYS_NOT_NULLABLE'],
+
```

Comment:
we need to register other data types. As follows
```suggestion
[['running_difference'], 'FLOAT', ['FLOAT'],
        '',
        '', '', 'vec', 'ALWAYS_NOT_NULLABLE'],```